opentreemap/treemap/units.py

```diff
@@ -31,6 +31,10 @@ def terminology(cls, instance=None):
                          .get(cls.__name__, {}))
         return terms
 
+    @classmethod
+    def display_name(cls, instance):
```

Comment:
As we discussed, I'd like to leave this in as a helper method, to make it easier to get the singular name.